cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/worker/BasicWorkerContext.java

```diff
@@ -143,18 +147,22 @@ public Metrics getMetrics() {
   }
 
   public LoggingContext getLoggingContext() {
-    //TODO: Add adapter name if present later
     return new WorkerLoggingContext(program.getNamespaceId(), program.getApplicationId(), program.getId().getId(),
-                                    getRunId().getId(), String.valueOf(getInstanceId()), null);
+                                    getRunId().getId(), String.valueOf(getInstanceId()), adapterName);
   }
 
   private static MetricsCollector getMetricCollector(MetricsCollectionService service, Program program,
-                                                     String runId, int instanceId) {
+                                                     String runId, int instanceId, @Nullable String adapterName) {
```

Comment:
there are some other nullable params here. not introduced by your PR, but lets make it consistent and mark them as well?